app/src/main/java/com/github/se/icebreakrr/model/profile/ProfilesViewModel.kt

```diff
@@ -186,6 +212,47 @@ open class ProfilesViewModel(
         onFailure = { e -> handleError(e) })
   }
 
+  /**
```

Comment:
Just an idea of implementation: when you select a picture on your phone, you might want to be able to crop the part of the picture you want to rather than the square center of the picture. Libraries like [Android Image Cropper](https://github.com/CanHub/Android-Image-Cropper) seem to allow this. You could try to implement this in M3.

Reply:
Yes, This was planned, but I didn't know how. I'll take a look at your recommendation at that moment. Thanks!